bbq/vm/test/vm_test.go

```diff
@@ -8591,14 +8591,17 @@ func TestEnumLookupFailure(t *testing.T) {
 
```

Comment:
Can you please add this change as a new test and keep the existing one? That way we test both scenarios